### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by merrcury**
Reformatted README for better usability. [Installing Gateway => Installing Environment => Running Code]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,19 +9,122 @@ Using a GRPC gateway is better than a REST gateway for the following reasons:
...
+    --restart always \
+    -e PRAVEGA_CONTROLLER \
+    -p 54672:80 \
+    --name pravega-grpc-gateway \
+    ${DOCKER_REPOSITORY}/pravega-grpc-gateway:${IMAGE_TAG}
+    ```
+
+### Running in Dell EMC Streaming Data Platform (SDP)
+
+1. Edit the file `charts/pravega-grpc-gateway/values.yaml` as needed. You will need to set `pravega.controlle`r and `service.annotations.external-dns.alpha.kubernetes.io/hostname` to match your environment.
```
Comment (claudiofahey) [inline README.md:63]: Typo at  `pravega.controlle`r.

**Review by claudiofahey — CHANGES_REQUESTED** [review]
Looks great! Please fix one typo and then I will merge this in.

File: README.md (hunk 1 above)
Comment (merrcury) [reply README.md]: Resolved with 9e53ca51f4aec15ef282a2da00c7a3086d672190

**Review by claudiofahey — APPROVED** [review]
Looks great. Thanks for the contribution!

**Pull request merged**